code/src/nuvla/connector/docker_compose_cli_connector.py

```diff
@@ -180,7 +180,7 @@ def sanitize_command_output(output):
         return '\n'.join(new_output)
 
     def _stack_services(self, project_name, docker_compose_path):
-        cmd = self.build_cmd_line(['-f', docker_compose_path, 'config', '--services', '--no-interpolate'], local=True)
+        cmd = self.build_cmd_line(['-f', docker_compose_path, 'config', '--services'], local=True)
```

Comment:
Right. This is why I updated the `execute_cmd` function to separate the stdout and stderr. To avoid having them merged and each time have to filter lines of text based

Reply:
so that goes into STDERR? ok then, in that case it should be ok